src/main/java/fact/datacorrection/DrsTimeCalibration.java

```diff
@@ -41,11 +41,13 @@
     SourceURL url = new SourceURL(DrsTimeCalibration.class.getResource("/long_term_constants_median.time.drs.fits"));
 
 
-    @Parameter(required = false, description = "")
+    @Parameter(required = false, description = "The name of the key, where the finished time calibrated data items are inserted.")
     String outputKey = "DataCalibrated";
 
+    @Parameter(required = false, description = "The name of the key with the startcelldata")
```

Comment:
Out of curiosity: why do you need to define the startcell key?

Reply:
Because i need to calibrate the pedestal and it has the key ped_startcelldata